src/ethereum/shanghai/fork.py

```diff
@@ -708,14 +717,18 @@ def calculate_intrinsic_cost(tx: Transaction) -> Uint:
     -------
     verified : `ethereum.base_types.Uint`
         The intrinsic cost of the transaction.
+    verified : `ethereum.base_types.Uint`
+        The calldata tokens used.
     """
     data_cost = 0
-
+    zb = 0
```

Comment:
We generally prefer that identifiers be fully spelled out. What does `zb` represent?

Reply:
I was refering to zero-bytes. Agree that it makes sense to spell it out, fixed it.